src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.yml

```diff
@@ -0,0 +1,10 @@
+name: ASP.NET Razor
+scopeName: text.aspnetcorerazor
+patterns:
+  - include: '#escaped-transition'
+  - include: 'text.html.basic'
+
+repository:
+  escaped-transition:
+    name: constant.character.escape.razor.transition
+    match: '@@'
```

Comment:
New line

Reply:
Will cover this in an upcoming PR to avoid mass rebases